Firestore/Source/Auth/FSTCredentialsProvider.mm

```diff
@@ -141,10 +119,9 @@ - (void)getTokenForcingRefresh:(BOOL)forceRefresh
             NSError *cancelError = [NSError errorWithDomain:FIRFirestoreErrorDomain
                                                        code:FIRFirestoreErrorCodeAborted
                                                    userInfo:errorInfo];
-            completion(nil, cancelError);
+            completion({"", User::Unauthenticated()}, cancelError);
```

Comment:
Two minor nits:

1. I don't know what `{"", User::Unauth()}` is. I suppose I could go look at the signature for completion... but I'm pretty lazy. Maybe do this instead: `completion(Token{"", User::Unauth()}, nil)`;
2. Rather than relying on invalid values, you might consider adding a static Token::Invalid() factory method as well as a Token::IsInvalid() or similar instance method. (Or you could just use null like the obj-c code did.)

Both optional.

Reply:
done